src/types.go

```diff
@@ -76,14 +82,17 @@ type Holdings struct {
 }
 
 type VenueHoldings struct {
-	VenueTotal       Holdings `json:"venue_total"`
-	AddressPrincipal Holdings `json:"address_holdings"`
-	AddressRewards   Holdings `json:"address_rewards"`
+	InfoMissing      bool      `json:"info_missing"`
```

Comment:
What are the semantics of the InfoMissing flag? should it just be expressed by the fields being null?

Reply:
Agreed that `nil`s would carry the same semantics (thought to do it that way as well), but it is more explicit this way.